docs/manual/hacking.rst

```diff
@@ -57,6 +65,8 @@ In practice, the development cycle looks something like this:
 #. run entire test suite: ``pytest``
 #. commit
 
+You don't need to run the xephyr script in order to run the tests.
```

Comment:
This is what I initially thought, so it might be worth to note.

Reply:
You could note that the test runner will launch its own x instances.